Sources/Fullscreen/LoginEntryView/LoginEntryDialogueView.swift

```diff
@@ -85,35 +85,39 @@ class LoginEntryDialogueView: UIView {
 
     private func setup() {
         layer.cornerRadius = 33
+        layoutMargins = UIEdgeInsets(
+            top: .veryLargeSpacing, leading: .largeSpacing, bottom: .largeSpacing, trailing: .largeSpacing
+        )
```

Comment:
The docs specify to use `directionalLayoutMargins` for iOS 11+ though, should we be using `layoutMargins`?

Reply:
https://developer.apple.com/documentation/uikit/uiview/1622566-layoutmargins

Reply:
that's a good point